book.rb

```diff
@@ -0,0 +1,16 @@
+class Book
+  attr_accessor :title, :author
```

Comment:
- [ ] Please make sure to add `rentals` also in order you will have access to it